README.md

```diff
@@ -139,6 +139,7 @@ The following table shows the supported combinations:
 
 | LiT               | LND          | Loop         | Faraday       | Pool         | Taproot Assets |
```

Comment:
We'd also need to add an entry for `0.10.2-alpha` above for minimum compatibility version of LND in remote mode.